src/uniffi_types.rs

```diff
@@ -345,3 +346,61 @@ impl UniffiCustomTypeConverter for NodeAlias {
 		obj.to_string()
 	}
 }
+
+/// Represents fee rate.
+///
+/// This is a simple wrapper around [`bitcoin::FeeRate`], only used for UniFFI bindings.
+#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+pub struct FeeRate(pub(crate) bitcoin::FeeRate);
```

Comment:
Could you remind me what prevents using `bitcoin::FeeRate` directly in bindings?

Reply:
> Hmm, I'm not sure I fully understand the proposal, but AFAIU this would complicate things even further.

Basically, add a trait that has a `from_sat_per_vb` method and implement it for `FeeRate` when `#[cfg(feature = "uniffi")]`.

> I think I'm leaning towards using `bitcoin::FeeRate` in bindings, i.e., leave current HEAD state. Let me know if you'd be fine with that and if I can squash.

SGTM